packages/form-layout/src/vaadin-form-item-mixin.js

```diff
@@ -6,8 +6,6 @@
 import { addValueToAttribute, removeValueFromAttribute } from '@vaadin/component-base/src/dom-utils.js';
 import { generateUniqueId } from '@vaadin/component-base/src/unique-id-utils.js';
 
-let itemRowSpacingDeprecationNotified = false;
```

Comment:
Removing the flag causes multiple deprecation warnings, one for each `vaadin-form-item`. In this case, I set the CSS variables to the `vaadin-form-layout` parent:
![image](https://github.com/user-attachments/assets/9253934d-a1a3-48ae-83e6-04f55f42e979)

Reply:
Oh, right. Let me revert it.

Reply:
Done.

Reply:
This might be getting a bit more complicated, but I guess we would need a flag for each property individually. Suppose you only set one of the deprecated properties in the `vaadin-form-layout` and another one in some of the `vaadin-form-item` elements (with the exception of the first one). In that case, the deprecation warning for the CSS variable set on the `vaadin-form-item` will not be shown.

Reply:
Good point!

Reply:
Done.